cmd/e2e-test/main.go

```diff
@@ -70,9 +70,9 @@ func main() {
 		stateTransitionZKVerifierAddress = flag.String("stateTransitionZKVerifierAddress", "", "state transition zk verifier smart contract address")
 		resultsZKVerifierAddress         = flag.String("resultsZKVerifierAddress", "", " results zk verifier smart contract address")
 		testTimeout                      = flag.Duration("timeout", 20*time.Minute, "timeout for the test")
-		sequencerEndpoint                = flag.String("sequencerEndpoint", defaultSequencerEndpoint, "sequencer endpoint")
+		sequencerEndpoint                = flag.StringSlice("sequencerEndpoint", []string{}, "sequencer endpoint(s)")
```

Comment:
I think that now `sequencersEndpoints` is more accurate since now it is a slice.

Reply:
if you mean the var name, i agree
the flag would stay being `e2etest --sequencerEndpoints http://blah --sequencerEndpoints http://bleh`